playground/nuxt.config.ts

```diff
@@ -8,7 +8,8 @@ export default defineNuxtConfig({
   tailwindcss: {
     // viewer: false,
     exposeConfig: true,
-    injectPosition: 'last'
+    injectPosition: 'last',
+    quiet: true
```

Comment:
could we default this to `nuxt.options.logLevel === 'silent'`?

https://github.com/nuxt/nuxt/blob/4e32e70bba8ad2500fb613206b6cce340d32fecf/packages/schema/src/config/build.ts#L42-L57